cli/tsc/98_lsp.js

```diff
@@ -312,8 +312,19 @@ function createLs() {
   return ls;
 }
 
+/** @param sourceFile {ts.SourceFile} */
+function isClassicScript(sourceFile) {
+  let isClassicScript = false;
+  let scriptSnapshot = SCRIPT_SNAPSHOT_CACHE.get(sourceFile.fileName);
+  if (!scriptSnapshot) {
+    return false;
+  }
+  return scriptSnapshot.isClassicScript ?? false;
+}
+
 /** @param {boolean} enableDebugLogging */
 export async function serverMainLoop(enableDebugLogging) {
+  ts.deno.setIsClassicScript(isClassicScript);
```

Comment:
Instead of modifying the ts compiler, does it work if we instead set `externalModuleIndicator` to undefined on the source file after parsing? (after calling `ts.createLanguageServiceSourceFile`?) Or is that set somewhere else beyond our control?

Reply:
Yes it works